ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/tenant_evpn_vxlan/leaf-tenant-igmp-snooping.j2

```diff
@@ -0,0 +1,37 @@
+{# Leaf tenant IGMP snooping #}
```

Comment:
IGMP snooping is a L2 functionality, so we should supportthis for both SVIs and L2 VLANs and implement on both L2LEAF and L3LEAF.

Reply:
Activated with commit aae1988